instance/tests/fixtures/api/appserver_detail.json

```diff
@@ -18,7 +18,8 @@
     "edx_platform_commit": "073e9c56a8093691dea782c8b727d839260833b1",
     "openedx_release": "master",
     "use_ephemeral_databases": true,
-    "github_admin_organization_name": "open-craft",
+    "github_admin_organizations": "[\"open-craft\"]",
+    "github_admin_users": "[]",
```

Comment:
The `JSONField`s are internally represented as `CharField`s, and the REST framework passes them on as strings, so it's correct the way it is.